adobe/react-spectrum
# fix(#2949): TableView/ListView/DnD Windows High Contrast issues

Relates to #2949 

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue 2949](https://github.com/adobe/react-spectrum/issue/2949).

## 📝 Test Instructions:

1. Using Google Chrome or Microsoft Edge, open https://reactspectrum.blob.core.windows.net/reactspectrum/5d6f3c03ccdf2462277f32e53624cceea767e751/storybook/iframe.html?providerSwitcher-express=false&providerSwitcher-toastPosition=bottom&strict=false&providerSwitcher-locale=&providerSwitcher-theme=&providerSwitcher-scale=&args=&id=tableview--allowsresizing-uncontrolled-sortable-columns&viewMode=story
2. Open Developer Tools, and enable `forced-colors: active` emulation under the Customize and control DevTools menu button (looks like "⋮" in the upper right corner of DevTools panel toolbar) > More tools... > Rendering > Emulate CSS media feature forced-colors > forced-colors: active
3. Use keyboard to navigate rows and cells in the TableView. Confirm that a visual indication of focus is present in forced-colors mode.
4. Select one or more rows. Confirm that it is possible to distinguish between selected and unselected rows, as well as the focused state of selected rows.
5. Resize columns to verify that the resize column indicator is visible 

<img width="968" alt="Screen capture of TableView in forced-color: active mode that demonstrates several rows selected with one of the selected rows focused" src="https://user-images.githubusercontent.com/154077/206735664-6741b5b7-44ec-46b2-ba9e-662193cab1d5.png" />

<img width="968" alt="Screen capture of TableView in forced-color: active mode that shows the Nubbin for resizing the column width of the first column" src="https://user-images.githubusercontent.com/154077/206736942-cf34f930-a3df-452d-825d-29e5baa01dca.png" />

Also test ListView stories, including drag and drop: https://reactspectrum.blob.core.windows.net/reactspectrum/e23100ba3eff9c6206f91b525d06f5620eedb5d4/storybook/index.html?path=/story/listview--default&providerSwitcher-express=false&providerSwitcher-toastPosition=bottom

And Drag and Drop stories:
https://reactspectrum.blob.core.windows.net/reactspectrum/5d6f3c03ccdf2462277f32e53624cceea767e751/storybook/index.html?path=/story/drag-and-drop--default&providerSwitcher-express=false&providerSwitcher-toastPosition=bottom

Checkbox WHCM story:
https://reactspectrum.blob.core.windows.net/reactspectrum/5d6f3c03ccdf2462277f32e53624cceea767e751/storybook/index.html?path=/story/checkbox--whcm&providerSwitcher-express=false&providerSwitcher-toastPosition=bottom

Switch WHCM story:
https://reactspectrum.blob.core.windows.net/reactspectrum/5d6f3c03ccdf2462277f32e53624cceea767e751/storybook/index.html?path=/story/switch--whcm-test&providerSwitcher-express=false&providerSwitcher-toastPosition=bottom

## 🧢 Your Project:

Adobe/Accessibility
